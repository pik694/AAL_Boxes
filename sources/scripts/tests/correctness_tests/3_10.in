3
60 25 21
29 86 89
34 41 82

