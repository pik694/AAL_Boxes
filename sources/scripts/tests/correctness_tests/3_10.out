0 2 1 
367774->336274
